repository_service_tuf_api/tasks.py

```diff
@@ -120,12 +122,16 @@ def get(task_id: str) -> Response:
         ``Response`` as BaseModel from pydantic
     """
     task = repository_metadata.AsyncResult(task_id)
+
+    task_state = task.state
+    task_result = task.result
+
     if isinstance(task.result, Exception):
         task_result = str(task.result)
-    else:
-        task_result = task.result
+    elif task_state == TaskState.SUCCESS and not task_result["status"]:
```

Comment:
```suggestion
    elif task_state == TaskState.SUCCESS and not task_result.get("status", False):
```

Reply:
Good @kairoaraujo, thank you for your suggestion. I'm changing now.